BUILD.XtraBackup.txt

```diff
@@ -10,14 +10,13 @@ Percona XtraBackup from source. These might vary from system to system.
 
 In Debian-based distributions, you need to:
 
-  $ apt-get install build-essential flex bison automake autoconf bzr \
-    libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev \
-    libgcrypt11-dev
+  $ apt-get install build-essential flex bison cmake libaio-dev \
+    mysql-client libncurses-dev zlib1g-dev libgcrypt11-dev
 
-In RPM-based distributions, you need to:
+In RPM-based distribution like CentOS 7, you need to:
```

Comment:
Slightly changed that. The version number was because older versions have a different package name for ncurses.